tests/TestCase/Core/FunctionsGlobalTest.php

```diff
@@ -332,11 +332,11 @@ public function testTriggerWarningEnabled(): void
     /**
      * Test no error when warning level is off.
      */
-    public function testTriggerWarningLevelDisabled(): void
-    {
-        $this->withErrorReporting(E_ALL ^ E_USER_WARNING, function (): void {
-            triggerWarning('This was a mistake.');
-            $this->assertTrue(true);
-        });
-    }
+    // public function testTriggerWarningLevelDisabled(): void
```

Comment:
This test generates:

```
1) /home/runner/work/cakephp/cakephp/src/Core/functions.php:315
This was a mistake.

Triggered by:

* Cake\Test\TestCase\Core\FunctionsGlobalTest::testTriggerWarningLevelDisabled
  /home/runner/work/cakephp/cakephp/tests/TestCase/Core/FunctionsGlobalTest.php:335
```

Not sure how to fix it.

Reply:
For now took care of this by not failing phpunit for warnings from the `globalfunctions` testsuite.